docs/source/contributing.rst

```diff
@@ -76,12 +76,12 @@ Create a `conda <https://docs.conda.io>`_ environment for echopype development
 
 .. code-block:: bash
 
-    conda create -c conda-forge -n echopype --yes python=3.9 --file requirements.txt --file requirements-dev.txt
+    conda create -c conda-forge -n echopype --yes python=3.9 --file requirements-dev.txt
     conda activate echopype
     # ipykernel is recommended, in order to use with JupyterLab and IPython
     # to aid with development. We recommend you install JupyterLab separately
     conda install -c conda-forge ipykernel
-    pip install -e .
+    pip install -e .[all]
```

Comment:
I think we should have some comments here to let users know (and as a reminder for ourselves) that the core requirements will be installed at this time, otherwise this can get confusing for folks seeing it for the first time.

Also some pointers for contributors know what `[all]` does.

Reply:
I agree. But, based on our comments below, it looks like we're converging on retaining `requirements.txt` w/o duplicating that dependency listing in `setup.cfg`. In that case, this additional explanation/comments won't be needed.